### PR TITLE
Copy update for 2018

### PR DESCRIPTION
**Pull request by Irio**
Still missing review in FAQ and About pages. For a following pull request.

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -1,26 +1,24 @@
...
-support:
+  faq: FAQ
+  in_other_language: "[Em Português]"
   fund_us: Support the project
 home:
-  about: Artificial Intelligence for social control of public administration
-  about_rosie: Operação Serenata de Amor created <a href="https://twitter.com/RosieDaSerenata" class="purple">Rosie</a>  - An artificial intelligence capable of analysing each claim for reimbursement from congresspeople and of identifying it’s illegal probability.
-  last_stories: Last stories
+  tagline: Artificial Intelligence for social control of public administration
+  about: Um projeto aberto que usa ciência de dados (e as mesmas tecnologias utilizadas por gigantes como Google, Facebook e Netflix) para fiscalizar gastos públicos e compartilha as informações de forma acessível a qualquer pessoa.
```
Comment (Irio) [inline _i18n/en.yml]: @cuducos Can you add an English version?

File: _i18n/en.yml [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
+  rosie: "A Serenata criou a <a href='https://twitter.com/RosieDaSerenata' class='purple'>Rosie</a>: uma inteligência artificial capaz de analisar cada um dos gastos de deputados federais e senadores feitos em exercício de sua função, identificando suspeitas e incentivando a população a questioná-los."
```
Comment (Irio) [inline _i18n/en.yml]: @cuducos Can you add an English version?

File: _i18n/en.yml [hunk 3]
```diff
@@ -1,26 +1,24 @@
...
+  jarbas: "Para que a população consiga entender a informação gerada pela Rosie, foi criado o <a href='https://jarbas.serenata.ai/' class='purple'>Jarbas</a>. Um site onde é possível navegar pelos gastos e descobrir mais sobre cada suspeita. O trabalho de validar uma suspeita começa nele."
```
Comment (Irio) [inline _i18n/en.yml]: @cuducos Can you add an English version?

File: _config.yml [hunk 4]
```diff
@@ -1,34 +1,38 @@
...
+  style: :compressed
 
 # About/contact
 author:
-  name:              Data Science Brigade
-  url:               https://www.facebook.com/operacaoSerenataDeAmor
-  email:             contato@datasciencebr.com
+  name: Operação Serenata de Amor
+  url: https://www.serenatadeamor.org
+  email: op.serenatadeamor@gmail.com
```
Comment (cuducos) [inline _config.yml:23]: `contato@serenata.ai`?

File: _config.yml [hunk 5]
```diff
@@ -1,34 +1,38 @@
...
+  sass_dir: _sass
+  style: :compressed
 
 # About/contact
 author:
-  name:              Data Science Brigade
-  url:               https://www.facebook.com/operacaoSerenataDeAmor
-  email:             contato@datasciencebr.com
+  name: Operação Serenata de Amor
+  url: https://www.serenatadeamor.org
```
Comment (cuducos) [inline _config.yml:22]: What about synchronizing and release in `https://serenata.ai` instead?

Comment (Irio) [reply _config.yml:22]: @cuducos serenata.ai is not working yet, so let's use .org from now.

File: _config.yml (hunk 4 above)
Comment (Irio) [reply _config.yml:23]: Can we wait the official release of the new domain? I couldn't find this email in any public and official page.

Comment (cuducos) [reply _config.yml:23]: > I couldn't find this email in any public and official page.

Cyclical, hum? If we never update our official pages, we will never find it in the official pages hahaha…

File: _config.yml (hunk 5 above)
Comment (cuducos) [reply _config.yml:22]: > serenata.ai is not working yet, so let's use .org from now.

What do you mean by _is not working_ yet? `jarbas.serenata.ai` is working. I think it's worse to have stuff in both. As we started to migrate, we should either continue to migrate or rollback whatever was migrated.

I'll stay with your _now_, but if you disagree with `serenata.ai` please let's talk about it.

Comment (Irio) [reply _config.yml:22]: jarbas.serenata.ai is working, not serenata.ai.

File: _i18n/en.yml (hunk 1 above)
Comment (cuducos) [reply _i18n/en.yml]: Done in a29de4c

File: _i18n/en.yml (hunk 2 above)
Comment (cuducos) [reply _i18n/en.yml]: Done in a29de4c

File: _i18n/en.yml (hunk 3 above)
Comment (cuducos) [reply _i18n/en.yml]: Done in a29de4c

**Pull request merged**